src/checkout/checkout_service.bal

```diff
@@ -0,0 +1,182 @@
+import ballerina/grpc;
+import ballerina/uuid;
+import ballerina/log;
+
+listener grpc:Listener ep = new (9094);
+
+configurable string cartUrl = "localhost";
+configurable string catalogUrl = "localhost";
+configurable string currencyUrl = "localhost";
+configurable string shippingUrl = "localhost";
+configurable string paymentUrl = "localhost";
+configurable string emailUrl = "localhost";
```

Comment:
`localhost` is the host name, not the URL. Better to rename these variables or change the values.